RemoteIDModule/WiFi_TX.cpp

```diff
@@ -21,31 +21,38 @@ bool WiFi_NAN::init(void)
 
     //set MAC address
     esp_base_mac_addr_set(mac_addr);
-
-    wifi_config_t wifi_config {};
-
-    WiFi.softAP("");
+    
+    if (g.webserver_enable == 0) {
+		WiFi.softAP(g.wifi_ssid, g.wifi_password, g.wifi_channel, false, 0); //make it visible and allow only 0 connection
+	}
+	else {
+		WiFi.softAP(g.wifi_ssid, g.wifi_password, g.wifi_channel, false, 1); //make it visible and allow only 0 connection
+	}
+	       
+    /*wifi_config_t wifi_config {};
 
     esp_wifi_get_config(WIFI_IF_AP, &wifi_config);
 
-    wifi_config.ap.ssid_hidden = 1;
-    wifi_config.ap.channel = wifi_channel;
+    wifi_config.ap.ssid_hidden = 0; //make always visible
+    wifi_config.ap.channel = g.wifi_channel;
 
     if (esp_wifi_set_config(WIFI_IF_AP, &wifi_config) != ESP_OK) {
         return false;
-    }
+    }*/
     if (esp_wifi_set_bandwidth(WIFI_IF_AP, WIFI_BW_HT20) != ESP_OK) {
         return false;
     }
-    if (esp_read_mac(WiFi_mac_addr, ESP_MAC_WIFI_STA) != ESP_OK) {
+    /*if (esp_read_mac(WiFi_mac_addr, ESP_MAC_WIFI_STA) != ESP_OK) {
         return false;
-    }
+    }*/
```

Comment:
Same here. Why is this commented out?

Reply:
well I noticed after the commit I needed to clean up a bit, I have done that now.